package.json

```diff
@@ -22,25 +22,18 @@
     "Aadit M Shah <aaditmshah@aadit.codes> (https://aadit.codes/)"
   ],
   "scripts": {
-    "format": "prettier --write . && npm run lint -- --fix",
+    "format": "prettier --write . && eslint . --fix",
     "lint": "npm-run-all \"lint:*\"",
     "lint:eslint-docs": "npm run update:eslint-docs && git diff --exit-code",
-    "lint:js": "eslint --report-unused-disable-directives .",
+    "lint:js": "eslint --report-unused-disable-directives . && prettier --check .",
```

Comment:
This will not just check JS, it will check everything when it comes to `prettier`, right?

I think it should be a 'lint:prettier' script that's checked in parallel with eslint (through npm-run-all)

But also:

In the format step the two are applied sequentially and there's nothing that guarantees that the end result style is still prettier compliant after the ESLint fixes has been made – it's quite classic that the two disagrees.

So I'm not sure there should be a `prettier --check` at all unless we're 100% sure the two will agree on all the things?

Reply:
> This will not just check JS, it will check everything when it comes to `prettier`, right?

Yes



> I think it should be a 'lint:prettier' script that's checked in parallel with eslint (through npm-run-all)

I can split it out. This PR was originally from before the `lint:*` targets were added, and I kept the changes minimal when I rebased it



> In the format step the two are applied sequentially and there's nothing that guarantees that the end result style is still prettier compliant after the ESLint fixes has been made – it's quite classic that the two disagrees.

ESLint has been deprecating those style rules since 8.53. Looking at this project, it is using the plugin/config for Prettier, that could be removed and split out to the lint target if you'd prefer

Reply:
I pushed a separate commit that strips out the `eslint-*-prettier` parts and just uses the plain Prettier calls. I left it as a separate commit to make it easier to roll off, if that's not what you were thinking